### DISCUSSION TITLE
How to I install and build with Uno 4?

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#15060] [Q&A] [answered]
I've got an app written in Uno4 but seem to be stuck with installing Uno5. Is there a way to install Uno4 templates in Visual Studio and have Uno4 as a build target?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. What do you mean by stuck?

If you want to create a template using an Uno 4 project, using :
```bash
dotnet new uninstall uno.templates
dotnet new install uno.templates::4.10.19
```

**Reply by axtens** [reply]
Stuck in the sense that I couldn't see a path to v4. All I could see was the the path to v5.

Thank you for your help.
